srcpkgs/python3-seaborn/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'python3-seaborn'
+pkgname=python3-seaborn
+version=0.11.0
+revision=1
+wrksrc="seaborn-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools python3-numpy python3-scipy python3-matplotlib python3-pandas"
```

Comment:
Drop `python3-setuptools` From `depends`; it is not a runtime depency.

Reply:
Done